sphinx_scality/static/css/theme.css

```diff
@@ -3,3 +3,122 @@
 
 @import url("./fonts.css");
 @import url("./current.css");
+
+
+/****** header******/
+
+div.header {
+    margin: 0 8% 0 7%;
+}
+
+
+
+
+/**** Header logo arrow search *****/
+
+img.baselogo {
+    width: 125px;
+}
+div.baselogo > a {
+  
+    font-size: 23px;
+    
+   
+}
+div.header > div.back-arrow img {
+    height: 30px;
+}
+
+/**** log ring****/
+
+div.logo img {
+    width: 133px;
+}
+
+
+
+/*****content******/
+
+div.content.inside {
+    margin: 0% 5% 0 10% !important;
```

Comment:
Why do we need `!important` here? It really is a [bad practice](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#The_!important_exception), and since we own the HTML, we just need to bump the specificity or use another selector that doesn't collide with other rules.